src/libraries/Common/src/Interop/Windows/Ole32/Interop.CreateStreamOnHGlobal.cs

```diff
@@ -8,7 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Ole32
     {
-        [DllImport(Libraries.Ole32)]
-        internal static extern int CoGetObjectContext([MarshalAs(UnmanagedType.LPStruct)] Guid riid, out IntPtr ppv);
+        [DllImport(Libraries.Ole32, PreserveSig = false)]
+        internal static extern IStream CreateStreamOnHGlobal(IntPtr hGlobal, bool fDeleteOnRelease);
```

Comment:
Took me a sec to figure out what was going on here.  Thanks for fixing the swap.

Reply:
I hear you. I was in similar position before :)